app/views/items/edit.html.erb

```diff
@@ -2,4 +2,51 @@
 
 <%= link_to "Back to list", list_path(@list), class: "pull-right" %>
 
-<%= render "form", url: item_path(list_id: params[:list_id]) %>
+<%= form_for @item, url: item_path(list_id: params[:list_id]) do |f| %>
```

Comment:
note to reviewer: this is just moved from a partial to this view. This is because the `new.html.erb` view is no longer and therefore the partial is unnecessary since this is the only view that will use it.